clean-lumber.lic

```diff
@@ -21,10 +21,20 @@ class CleanLumber
     args = parse_args(arg_definitions)
     settings = get_settings
     tool_storage = settings.crafting_container
+    @bag = settings.crafting_container
```

Comment:
What is the difference between `tool_storage` and `@bag`?

Reply:
looking at it now, nothing. tool_storage was what was in there. I added the known pieces to make tool belts work. I can take tool_storage out and change any place referenced to @bag.

Reply:
Testing the changes.